packs/bestiary-effects/effect-field-of-past-regrets.json

```diff
@@ -0,0 +1,58 @@
+{
+    "_id": "y6fibVDs9QF2vbwu",
+    "img": "icons/magic/unholy/orb-swirling-pink.webp",
+    "name": "Effect: Field of Past Regrets",
+    "system": {
+        "description": {
+            "value": "<p>Creatures who have at least one past life gain a +1 status bonus to saving throws against mental effects. Living, non-nindoru creatures gain weakness 3 to mental damage.</p>"
```

Comment:
Not sure if this is a spoiler. If not, we could also add a RollOption toggle.

Reply:
Just have it always grant the +1